geozero/src/error.rs

```diff
@@ -37,6 +40,9 @@ pub enum GeozeroError {
     // General
     #[error("I/O error")]
     IoError(#[from] std::io::Error),
+    #[cfg(feature = "with-mvt")]
+    #[error("MVT error")]
+    MvtError(#[from] MvtError),
```

Comment:
This way you can delete the conditional use statement above.

```suggestion
    MvtError(#[from] crate::mvt::mvt_error::MvtError),
```